manifests/repomd_signatures.pp

```diff
@@ -0,0 +1,21 @@
+# Generate or regenerate Yum repository metadata GPG signatures
+class pulp::repomd_signatures {
+  if !$::pulp::yum_regenerate_repomd_signatures {
+    $existing_sig_test = '[ -e repomd.xml.asc ] || '
+  }
+  $regen_command = "/usr/bin/sudo -u apache -H /usr/bin/bash -c '
+    cd /var/lib/pulp/published/yum/master/yum_distributor/ 2>/dev/null || exit 0
+    for d1 in `ls` ; do
```

Comment:
Can we use `*` instead of relying on `ls`?

Reply:
Ok.